clans/src/main/java/me/mykindos/betterpvp/clans/clans/ClanManager.java

```diff
@@ -92,6 +92,14 @@ public class ClanManager extends Manager<Clan> {
     @Config(path = "clans.dominance.enabled", defaultValue = "true")
     private boolean dominanceEnabled;
 
+    @Inject
+    @Config(path = "clans.dominance.fixedDominanceGain", defaultValue = "true")
+    private boolean fixedDominanceGain;
+
+    @Inject
+    @Config(path = "clans.dominance.dominanceGain", defaultValue = "5.0")
+    private double dominanceGain;
+
```

Comment:
Can you change this to be

clans.dominance.fixed.enabled and clans.dominance.fixed.delta

Config change would just be

```
dominance:
    enabled: true
    fixed:
        enabled: true
        delta: 5.0